rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -548,14 +548,14 @@ void RSXThread::update_reg(u32 reg, u32 value)
 
 	case NV4097_SET_CULL_FACE:
 	{
-		m_cull_face = value;
+		m_cull_face = (value == 0x0404) || (value == 0x0405) || (value == 0x0408) ? value : 0x0405/*default GL_BACK*/;
```

Comment:
Alright. may be i keep this unchange at this moment since i didn't find invalid value passing to glcullface yet but only glfrontface.